### PR TITLE
Vpp agent

### PR DESCRIPTION
**Pull request by filvarga**
Added strongswan debug logging configuration for dev env.
Improved build/install scritpts.
Fixed ubuntu 18.04 building issues.

### CODE REVIEW

File: build_all.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 set -e
 
+LOG="/tmp/build.log"
```
Comment (mestery) [inline build_all.sh]: I like this idea, but this file will grow quite huge, is it possible to either roll It over with the start of each run, or just delete it at the start of a run.

File: build_dependencies.sh [hunk 2]
```diff
@@ -18,27 +18,43 @@
 set -e
 
 WS="`pwd`"
+LOG="/tmp/build.log"
```
Comment (mestery) [inline build_dependencies.sh]: Same logging question as above here.

File: scripts/dev-env.sh [hunk 3]
```diff
@@ -155,7 +173,7 @@ start() {
   vpp_conf
 
   echo "info: starting docker containers"
-  (sudo docker run --name responder --hostname responder -d --net=host --privileged -it -e INITIAL_LOGLVL=debug -e ETCD_CONFIG=DISABLED -e KAFKA_CONFIG=DISABLED -v $VPP_CFG_DIR:/etc/vpp -v $AGENT_CFG_DIR:/opt/vpp-agent/dev ligato/vpp-agent:pantheon-dev && sudo docker run --name initiator --hostname initiator -d --privileged -v $INITIATOR_CFG_DIR:/conf -v $INITIATOR_CFG_DIR:/etc/ipsec.d philplckthun/strongswan) &> /dev/null
+  (sudo docker run --name responder --hostname responder -d --net=host --privileged -it -e INITIAL_LOGLVL=debug -e ETCD_CONFIG=DISABLED -e KAFKA_CONFIG=DISABLED -v $VPP_CFG_DIR:/etc/vpp -v $AGENT_CFG_DIR:/opt/vpp-agent/dev ligato/vpp-agent:pantheon-dev && sudo docker run --name initiator --hostname initiator -d --privileged -v $INITIATOR_CFG_DIR:/conf -v $INITIATOR_CFG_DIR:/etc/ipsec.d philplckthun/strongswan) 1> /dev/null
```
Comment (mestery) [inline scripts/dev-env.sh]: Looks like we're running the Docker image from your DockerHub, is that correct? What is in the `philplckthun/strongswan` Docker image, and should we be running what is built here by checking in the Dockerfile? I may not be understanding what is being run here, so apologies if that's the case.

Comment (filvarga) [reply scripts/dev-env.sh]: Docker image philplckthun/strongswan contains strongswan with default plugins. This is not my dockerhub image. We only do some basic network configuration and configure strongswan in this container.

File: build_all.sh (hunk 1 above)
Comment (filvarga) [reply build_all.sh]: It's not probably visible well enough. But the file is cleared running first stdout redirect in build_dependencies.sh.

Comment (filvarga) [reply build_all.sh]: I have changed the creation of the log file so it is more clear.

**Pull request merged**